nml/actions/action0properties.py

```diff
@@ -818,6 +819,36 @@ def station_tile_flags(value):
     ]
 
 
+def array_for_station_properties(prop_num: int, list_type: str, prefered_size: int, invert_function: callable):
+    def generated_function(value):
+        if isinstance(value, ConstantNumeric):
+            value = Array([value, invert_function(value)] * prefered_size, value.pos)
```

Comment:
This is not a good idea to me.
All tiles having the same pillar placement limitation is probably quite rare.
And always defining 20 tile types feels wasteful.